cli/pkg/stack_mgr/stack.go

```diff
@@ -296,16 +296,6 @@ func (s *Stack) Apply(ctx context.Context, waitOptions options.WaitOptions, runO
 
 	logrus.Debugf("will use tf bundle found at %s", srcDir)
 
-	tempFile, err := os.CreateTemp("", "happy_tfe.*.tar.gz")
```

Comment:
This block of code was producing a gz file that wasn't used anywhere.

Reply:
I've seen this! I always assumed I didn't know what was going on